### ISSUE TITLE
Correções tutorial

### ISSUE THREAD

**Issue by NayaraCaetano** [ayr-ton/tutorial-djangogirls#1]
Ayrton, acho que o que gerou aqueles erros no pythonanywhere foi a versão do django
Para instalar local o comando do tutorial especifica a versão
Mas na hora do deploy ele só pede para executar pip install django

Acho que isso gerou a divergência nos erros que só aconteciam "em produção". Já que os settings usados eram os mesmos

Além do problema do allowed hosts tivemos também o problema da chamada da url dentro dos templates, pois não estava definido o name da url no urls.py

**Comment by ayr-ton** [comment]
@NayaraCaetano Sim, no python anywhere pedia pra instalar só django sem a versão, daí instalou a versão 1.11.1. No código tinha um TODO pra usar o requirements. Então fiz o seguinte aqui 9499cbae942

Revisa pra mim se puder? x)

**Comment by ayr-ton** [comment]
Perae que o gitbooks mexeu num monte de coisas. Deixa eu rodar um git reset antes.

**Comment by ayr-ton** [comment]
Pronto, o commit certo é 45d7912e
